shell/edit/fleet.cattle.io.gitrepo.vue

```diff
@@ -265,6 +289,16 @@ export default {
       ];
     },
 
+    showPollingIntervalWarning() {
```

Comment:
I can see why we need to warn the user of a polling interval lower than 15 seconds, however if it's not a hard requirement we shouldn't force scroll the user to view it

Reply:
This also doesn't take into account if `disablePolling` is true

Reply:
We can remove the scroll, the field will be totally visible when we will move the auth section to new steps: https://github.com/rancher/dashboard/issues/13402